lib/src/internal/http_endpoints.dart

```diff
@@ -1583,7 +1583,29 @@ class HttpEndpoints implements IHttpEndpoints {
       "${Constants.cdnUrl}/guilds/$guildId/users/$memberId/avatars/$avatarHash.$format";
 
   @override
-  String getUserBannerURL(Snowflake userId, String hash, {String format = "png"}) => "${Constants.cdnUrl}/banners/$userId/$hash.$format";
+  String? getUserBannerURL(Snowflake userId, String? hash, {String? format, int? size}) {
+    if (hash != null) {
```

Comment:
it could be inverted so if hash is null you return null and otherwise rest of logic. it would removed one level of nesting